packages/xod-client/src/project/components/Node.jsx

```diff
@@ -100,10 +106,11 @@ class Node extends React.Component {
           onMouseUp={this.onMouseUp}
           onDoubleClick={this.onDoubleClick}
           id={id}
-          title={nodeLabel} // this is for func-tests
+          data-label={nodeLabel} // for func tests
+          title={errTitle || nodeLabel}
         >
           {this.renderBody()}
-          {!this.props.isDragged ? <title>{nodeLabel}</title> : null}
+          {!this.props.isDragged ? <title>{errTitle || nodeLabel}</title> : null}
```

Comment:
Not sure `errTitle || nodeLabel` is a proper decision.

1. Code duplication with a line above
2. `nodeLabel` is inaccessible in case there are errors

Could we provide a better experience with rich tooltips which show the base name (if overriden) *and* the error (if present)?